scripts/analysis.py

```diff
@@ -1,47 +1,16 @@
-import sqlite3 as lite
-import sys
+import collections
 import numpy as np
 import matplotlib.pyplot as plt
+import sqlite3 as lite
+import sys
 from matplotlib import cm
-import collections
-import re
+from pyne import nucname
+
 
 if len(sys.argv) < 2:
     print('Usage: python analysis.py [cylus_output_file]')
 
-
-def snf(cursor):
-    """prints total snf and isotope mass
-
-    Parameters
-    ----------
-    cursor: cursor
-        cursor for sqlite3
-
-    Returns
-    -------
-    array
-        inventory of individual nuclides
-        in format nuclide = mass [kg]
-    """
-
-    cur = cursor
-
-    sink_id = get_agent_ids(cur, 'sink')
-
-    # get resources that ended up in sink.
-    resources = cur.execute(exec_string(sink_id,
-                                        'transactions.receiverId',
-                                        'qualid')).fetchall()
-
-    # get list of sum(quantity) and qualid for snf
-    snf_inventory = cur.execute(exec_string(sink_id,
-                                            'transactions.receiverId',
-                                            'sum(quantity), qualid')
-                                + ' group by qualid').fetchall()
-
-    waste_id = get_waste_id(resources)
-    return isotope_calc(waste_id, snf_inventory, cur)
+"""HELPERS"""
```

Comment:
This was added while reviewing the script to know which functions we had. It was mainly a change to help us review the overall code.

If this is a poor use of docstring, I will remove them.

Thank you :)